Detections/Syslog/ssh_NewlyInternetExposed.yaml

```diff
@@ -20,27 +21,28 @@ query: |
   let PrivateIPregex = @'^127\.|^10\.|^172\.1[6-9]\.|^172\.2[0-9]\.|^172\.3[0-1]\.|^192\.168\.'; 
   let avgthreshold = 0;
   let probabilityLimit = 0.01;
+  let startime = 7d;
   let ssh_logins = Syslog
-  | where TimeGenerated >= ago(7d)
-  | where Facility contains "auth" and ProcessName != "sudo"
+  | where TimeGenerated >= ago(startime)
+  | where Facility contains "auth" and ProcessName =~ "sshd"
   | where SyslogMessage has "Accepted"
   | extend SourceIP = extract("(([0-9]{1,3})\\.([0-9]{1,3})\\.([0-9]{1,3})\\.(([0-9]{1,3})))",1,SyslogMessage) 
   | where isnotempty(SourceIP)
-  | extend ipType = iff(SourceIP matches regex PrivateIPregex,"private" ,"public" );
+  | extend ipType = iff(SourceIP matches regex PrivateIPregex,"private" ,"public");
   ssh_logins 
   | summarize privatecount=countif(ipType=="private"), publiccount=countif(ipType=="public") by HostName, HostIP, bin(EventTime, 1d)
   | summarize 
-  publicIPLoginHistory  = make_list(pack('IPCount', publiccount,  'logon_time', EventTime)),
-  privateIPLoginHistory = make_list(pack('IPCount', privatecount, 'logon_time', EventTime)) by HostName, HostIP
+  publicIPLoginHistory  = makelist(pack('IPCount', publiccount,  'logon_time', EventTime)),
```

Comment:
make_list() is the function that should be used, please change this back, makelist() is the legacy/deprecated version - https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/makelist-aggfunction